src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -44,8 +47,17 @@ public void play() {
     waitForVideoStart();
   }
 
+  public void playOnFandom() {
+    runInAdFrame(() -> wait.forElementClickable(playTriggerButtonSelector).click());
+    waitForVideoStartOnFandom();
+  }
+
   public void pause() {
-    driver.findElement(By.className("overVideoLayer")).click();
+    driver.findElement(By.cssSelector(String.format(UI_ELEMENT_SELECTOR_FORMAT, slotName))).click();
+  }
+
+  public void pauseVideoOnFandom() {
+    driver.findElement(By.cssSelector(String.format(UI_ELEMENT_SELECTOR_ON_FANDOM_FORMAT, slotName))).click();
   }
 
   private interface Lambda {
```

Comment:
This is exactly the same interface as `Runnable`. Does it make sense to change and remove `Lambda`?